ext/pdo_mysql/tests/pdo_mysql_local_infile_directory_denied.phpt

```diff
@@ -40,14 +40,13 @@ if (!defined('PDO::MYSQL_ATTR_LOCAL_INFILE_DIRECTORY')) {
 	MySQLPDOTest::createTestTable($db, MySQLPDOTest::detect_transactional_mysql_engine($db));
 
 	try {
-		exec_and_count(1, $db, 'DROP TABLE IF EXISTS test', 0);
```

Comment:
a few similar `DROP TABLE` statements called via the `exec_and_count()` function were not removed beforehand